sections/difficulty.tex

```diff
@@ -26,8 +26,38 @@
 
 \clearpage
 
-\includegraphics[width=\linewidth]{\tables/optional_rules.png}
-
-\vspace*{\fill}
+\vspace*{-2em}
+\hommtable{37}{
+  \centering
+  \textbf{Optional Rules Table}
+
+  \small
+  {
+    \color{white}
+    You may modify the rules to increase or decrease the game's difficulty.
+  }
+  \bigskip
+
+  \begin{tabularx}{\linewidth}{>{\hsize=0.3\hsize\linewidth=\hsize}X
+                               >{\hsize=1.7\hsize\linewidth=\hsize}X}
+     \darkcell[1.5]{\color{white}Game Difficulty Levels} & \darkcell[1.5]{Change to the default rules} \\
```

Comment:
```suggestion
     \darkcell[1.5]{\color{white}Game Difficulty Levels} & \darkcell[1.5]{\textbf{Change to the default rules}} \\
```

Reply:
I'll instead put it into \darkcell itself